hasher-matcher-actioner/hmalib/common/tests/test_content_models.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+import unittest
+from contextlib import contextmanager
+from moto import mock_dynamodb2
+from hmalib.common.content_models import ContentObject, ActionEventRecord
+from hmalib.common.evaluator_models import ActionLabel, ActionRule
+from hmalib.common.message_models import BankedSignal, ActionMessage
+from hmalib.common.classification_models import ClassificationLabel
+
+
+import boto3
+import datetime
+import os
+
+
+class TestContentModels(unittest.TestCase):
+    table = None
+    TEST_CONTENT_ID = "test_content_id_1"
+    TEST_TIME = datetime.datetime(2021, 5, 17, 13, 38, 56, 965173)
+    TEST_ACTION_LABEL = "TestEnqueueMiniCastleForReview"
+
+    @staticmethod
+    def mock_aws_credentials():
+        """
+        Mocked AWS Credentials for moto.
+        (likely not needed based on local testing but just incase)
+        """
+        os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+        os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+        os.environ["AWS_SECURITY_TOKEN"] = "testing"
+        os.environ["AWS_SESSION_TOKEN"] = "testing"
+
+    @classmethod
+    def setUpClass(cls):
+        cls.mock_aws_credentials()
+        cls.mock_dynamodb2 = mock_dynamodb2()
+        cls.mock_dynamodb2.start()
+        cls.create_mocked_table()
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.mock_dynamodb2.stop()
+
+    @classmethod
+    def create_mocked_table(cls):
```

Comment:
At some point, we might want to abstract out common features for tests. 

I also like starting "afresh".  See: https://github.com/facebook/ThreatExchange/blob/master/hasher-matcher-actioner/hmalib/tests/test_pdq_models.py#L346-L353

This allows you to have a clean dynamodb using a context-manager. 

Note: Not saying that you need to do this here. This is good!

Reply:
Yes! thought the same thing when writing it: https://github.com/facebook/ThreatExchange/issues/594